### PR TITLE
enh(doc) add doc pp application netbackup ssh

### PR DESCRIPTION
**Pull request by psamecentreon**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+id: applications-netbackup-ssh
+title: Netbackup SSH
+---
+
+## Plugin Pack Assets
+
+### Monitored Objects
+
+The Centreon Plugin Pack includes monitoring of job status, tape usage, drive cleaning, and drvie status using system commands.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:10]:
```suggestion
The Centreon Plugin Pack includes monitoring of job status, tape usage, drive cleaning, and drive status using system commands.
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+| :---------------------------------------- | :------------------------------------------ |
+| App-Netbackup-Job-Per-Policy              | Discover Job  per policy                    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Collected Metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Dedupstatus -->
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:28]:
```suggestion
<!--Dedup-status -->
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+| Metric name                               | Description                            | Unit |
+| :---------------------------------------- | :------------------------------------- | :--- |
+| status                                    | Status of dedup                        |      |
+| usage                                     | Dedup usage                            |  %   |
+
+<!--Drivecleaning-->
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:35]:
```suggestion
<!--Drive-cleaning-->
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+| Metric name                               | Description                         | Unit  |
+| :---------------------------------------- | :---------------------------------- | :---  |
+| cleaning                                  | Total number of drive cleaning      | count |
+
+<!--Drivestatus-->
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:41]:
```suggestion
<!--Drive-status-->
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+| Metric name                               | Description                                                       | Unit  |
+| :---------------------------------------- | :---------------------------------------------------------------- | :---- |
+| status                                    | Status of drive            	                                    |       |
+
+<!--Jobstatus-->
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:47]:
```suggestion
<!--Job-status-->
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+| Metric name                               | Description                            | Unit  |
+| :---------------------------------------- | :------------------------------------- | :---- |
+| status                                    | Status of jobs                         |       |
+| total                                     | Total of jobs                          | count |
+| frozen                                    | Jobs in frozen state                   |       |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:53]: It's a count too no ?

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+<!--Tapeusage-->
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:55]:
```suggestion
<!--Tape-usage-->
```

File: en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+    --ssh-username=centreon \
+    --ssh-password='centreon-password' \
+    --ssh-backend=sshcli \
+    --critical-status='%{status} !~ /up/i' \ 
+    --verbose
+```
+
+The above command gets the state of the Symantec Netbackup jobs (```--mode=job-status```).
+It uses a SSH username _centreon_ (```--ssh-username=centreon```), a SSH password _centreon-password_ (```--ssh-password='centreon-password'```),
+uses a SSH backend _sshcli_ (```--ssh-backend='sshcli'```) and it connects to the host _10.30.2.81_ (```--hostname=10.30.2.81```).
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:166]: Explain the threshold here no ?

File: fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
+---
+id: applications-netbackup-ssh
+title: Netbackup SSH
+---
+
+## Contenu du Plugin Pack
+
+### Objets supervisés
+
+Le Plugin Pack inclue la supervision du statut tâches, l'utilisation des cassets, le nettoyage des devices, et le status des devices
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:10]:
```suggestion
Le Plugin Pack inclut la supervision du statut des tâches, l'utilisation des cassettes, le nettoyage des équipements, et le status des équipements
```

File: fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+| total                                     | Total of jobs                          | count |
+| frozen                                    | Jobs in frozen state                   |       |
+
+<!--Tapeusage-->
+
+| Metric name                               | Description                                             | Unit  |
+| :---------------------------------------- | :------------------------------------------------------ | :---  |
+| used                                      | Usage tape on the deivce                                |   %   |
+| free                                      | Free tape on the device                                 |   %   |
+| usage                                     | Total of tape usage                                     | count |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:62]: Same as english review

File: fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+    --ssh-username=centreon \
+    --ssh-password='centreon-password' \
+    --ssh-backend=sshcli \
+    --critical-status='%{status} !~ /up/i' \ 
+    --verbose
+```
+
+La commande ci-dessus contrôle le status des tâches Symantec Netbackup (```--mode=job-status```).
+Le Plugin utilise le Backend _sshcli_ (```--ssh-backend='sshcli'```) avec l'utisateur _centreon_ (```--ssh-username=centreon```), 
+son mot de passe (```--ssh-password='centreon-password'```) et il se connecte à l'hôte _10.30.2.81_ (```--hostname='10.30.2.81'```).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-netbackup-ssh.md:169]: Explain the threshold

**Review by pkriko — CHANGES_REQUESTED** [review]

**Review by Sims24 — APPROVED** [review]

**Comment by Sims24** [comment]
Replaced by https://github.com/centreon/centreon-documentation/pull/1195

**Pull request closed without merging**